ssh/DOCS.md

```diff
@@ -65,14 +72,22 @@ Specifies whether TCP forwarding is permitted or not.
 
 **Note**: _Enabling this option lowers the security of your SSH server! Nevertheless, this warning is debatable._
 
+
 ## Network
-To enable ssh access via the network, you need to enter the port number ‘22’ or the port you want to use. This will map that port from the hassio host into the running “Terminal & SSH” container.
+
+To enable ssh access via the network, you need to specify which external port to use. 
+The number you enter, will be used to map that port from the hassio host into the 
+running "Terminal & SSH" container.
+
+The standard port for the SSH protocol is `22`.
+
 
 ## Known issues and limitations
 
 - This add-on will not enable you to install packages or do anything as root.
   This is not working with Home Assistant.
 
+
```

Comment:
Not needed extra new line.

Reply:
The use of new lines in this documentation file was inconsistent. I normalized this so sections could be found by quickly scanning the document. I went with what I've seen before, which is that headings of level 1 and 2 (`#` and `##`) have two empty lines above them.

I've not seen the suggestion of a linter or similar, just that Black is used for source code - nothing around consistency of flow, ' vs *, ... in documentation. If there is expected formatting to be used, I suggest it is added to https://developers.home-assistant.io/docs/documenting/standards

Reply:
multiple new lines have no effect. please remove them as requested. Thanks.

Reply:
> multiple new lines have no effect

I disagree on whether white space and formatting affects readability of/ability to (quickly) consume a text, but can adapt.
I agree it makes not difference for the rendered documentation, but it makes a difference for the person that needs to maintain the source documentation.

In my opinion, you're prioritizing a program's processing aspects of a text, over humans' ability to process a text. There's a book titled Clean Code that has several examples of this, for source code. I'm arguing you're optimizing for the smaller cost in your project, rather than the by orders of magnitude larger cost. The highest cost for any change to a system, is almost always the process that involves a human interpreting text (maintaining the Markdown version), never the compiler/processor.

For future, I suggest this project adapt https://github.com/markdownlint/markdownlint and integrate it into the CI process.

Reply:
Thanks for the education. Markdownlint would not have agreed as well. Furthermore, markdownlint isn't the goto linter for Markdown anymore, instead, we use Remarklint, which is slowly being rolled out in our repositories.